tests/clims/api/endpoints/test_substances.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+
+import random
+import json
+
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+
+from tests.clims import testutils
+from rest_framework import status
+from clims.models import Substance
+
+
+class SubstancesTest(APITestCase):
+    def test_get_substances(self):
+        substance = testutils.create_substance('sample-001', {})
+
+        url = reverse('clims-api-0-substances', args=(substance.organization.name,))
+        self.login_as(self.user)
+        response = self.client.get(url)
+
+        assert response.status_code == 200, response.content
+        assert len(response.data) == 1
```

Comment:
I may have misunderstood this. But if another test creates a sample with with testutils.create_substance(), isn't this test going to fail then?

Reply:
Yes nice catch. It seems that these API test cases aren't transactional. I'll change this to just requiring one more than the previous run.